frappe/erpnext
# feat: copy emails from lead to customer

When creating a new **Customer** from a **Lead** and _Carry Forward Communication and Comments_ is enabled in **CRM Settings**, then copy communication from **Lead** to **Customer**.

How to test this:

1. Create a new **Lead**
2. Send an email from the **Lead** form (say, to the lead)
3. Klick on Create > Customer, save
4. The email should be visible in the **Customer**'s _Activity_

> no-docs

(will add after merge)
